File: VB/XtraPivotGridTutorial.Step4/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace XtraPivotGridTutorial
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			xtraPivotGridTutorialViewTableAdapter.Fill(nwindDataSet.XtraPivotGridTutorialView)
			pivotGridControl1.BestFit()
			pivotGridControl1.FieldsCustomization(splitContainerControl1.Panel1)
			pivotGridControl1.CollapseAll()
		End Sub

		Private Sub pivotGridControl1_ShowingCustomizationForm(ByVal sender As Object, ByVal e As DevExpress.XtraPivotGrid.CustomizationFormShowingEventArgs) Handles pivotGridControl1.ShowingCustomizationForm
			e.ParentControl = splitContainerControl1.Panel1
		End Sub
	End Class
End Namespace

File: CS/XtraPivotGridTutorial.Step4/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace XtraPivotGridTutorial
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            xtraPivotGridTutorialViewTableAdapter.Fill(nwindDataSet.XtraPivotGridTutorialView);
            pivotGridControl1.BestFit();
            pivotGridControl1.FieldsCustomization(splitContainerControl1.Panel1);
            pivotGridControl1.CollapseAll();
        }

        private void pivotGridControl1_ShowingCustomizationForm(object sender, DevExpress.XtraPivotGrid.CustomizationFormShowingEventArgs e)
        {
            e.ParentControl = splitContainerControl1.Panel1;
        }
    }
}
